From 3f65494a4cb2544eb16af3fa64a161bd8142f487 Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Tue, 13 Jan 2015 12:04:38 -0500
Subject: [PATCH] increase syslog message limit from 256 to 1024

this addresses alpine linux issue #3692 and brings the syslog message
length limit in alignment with uclibc's implementation.
---
 src/misc/syslog.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/misc/syslog.c b/src/misc/syslog.c
index fbce4bc..d587499 100644
--- a/src/misc/syslog.c
+++ b/src/misc/syslog.c
@@ -81,7 +81,7 @@ static void _vsyslog(int priority, const char *message, va_list ap)
 	char timebuf[16];
 	time_t now;
 	struct tm tm;
-	char buf[256];
+	char buf[1024];
 	int errno_save = errno;
 	int pid;
 	int l, l2;
-- 
2.3.3

